sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -184,6 +191,15 @@ for recognized_form in result:
         ))
 ```
 
+Alternatively, a form url can also be used to recognize custom forms using the `begin_recognize_custom_forms_from_url` method.
```

Comment:
Can we add a quick sentence about the `from_url` existing for all recognize methods?